### PR TITLE
Add kustomize application to deploy rag-llm application

### PR DESCRIPTION
**Pull request by sauagarwa**

### CODE REVIEW

File: common/.gitignore
Comment (claudiol) [inline common/.gitignore:1]: @sauagarwa If we are going to change this file we should do so directly in the **common** repository.  Remember that we use git subtree to include this repo in our patterns.

File: common/scripts/pattern-util.sh
Comment (claudiol) [inline common/scripts/pattern-util.sh:1]: @sauagarwa Same with this file.  It seems there are no real changes to this file ... If we are going to change this file we should do so directly in the **common** repository.  Remember that we use git subtree to include this repo in our patterns.

**Review by claudiol — CHANGES_REQUESTED** [review]
@sauagarwa Let's revert the changes to .gitignore and pattern-util.sh.  If we are going to change these files we should do so directly in the **common** repository.  Remember that we use git subtree to include this repo in our patterns.

File: common/.gitignore
Comment (sauagarwa) [reply common/.gitignore:1]: I will revert it back .. i think there is only spacing issue

Comment (sauagarwa) [reply common/.gitignore:1]: fixed

**Comment by claudiol** [comment]
Merging. This patterns needs nodes that include access to GPUs.  We will add that in the documentation of the pattern.

**Pull request merged**